<?xml version="1.0"?>
<launch>
    <arg name="bag_file" default="/home/derrick/Desktop/new_data/dynamic_global_globalization/ml_std_test2018-07-18-17-51-04/filtered_map_server_test_2018-07-18-17-13-24.bag"/>
    <arg name="dist_max" default="0.1" />
    <arg name="theta_max" default="0.1" />
    <arg name="freq_min" default="0.1" />
    <arg name="time_min" default="20" />
    <arg name="error_check_topics_str" default="/scan_with_pose;" />
    <arg name="miss_check_topics_str" default="/odom2D;/scan_with_pose;/scan;" />
    <node pkg="rosbag_error_checker" type="rosbag_error_checker" name="rosbag_error_checker" cwd="node" output="screen">
    <!-- full direcotry of the bag -->
    <param name="bag_file"  type="string"  value ="$(arg bag_file)" /> 
    <!--  maximum allowed translation (meter) of robot motion between two frames    -->
    <param name="dist_max"  type="double"  value = "$(arg dist_max)" /> 
    <!--  maximum allowed rotation (radius) of robot motion between two frames  -->
    <param name="theta_max" type="double"  value = "$(arg theta_max)" /> 
    <!-- minimum allowed frequency (Hz) between two frames. -->
    <param name="freq_min"  type="double"     value ="$(arg freq_min)" />
    <!-- minimum allowed duration (seconds) for a given topic to be listed exist.     -->
    <param name="time_min" type="double"     value = "$(arg time_min)" /> 
    <param name="error_check_topics_str" type="string" value="$(arg error_check_topics_str)" />
    <param name="miss_check_topics_str" type="string" value="$(arg miss_check_topics_str)" />


    </node>
</launch>
